include/slam_toolbox/experimental/slam_toolbox_lifelong.hpp

```diff
@@ -30,7 +30,12 @@ class LifelongSlamToolbox : public SlamToolbox
 public:
   explicit LifelongSlamToolbox(rclcpp::NodeOptions options);
   ~LifelongSlamToolbox() {}
+  rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn
+  on_configure(const rclcpp_lifecycle::State &) override;
```

Comment:
Where's activate and deactivate? Ditto across the PR

Reply:
If not overridden, `on_activate` and `on_deactivate` in SlamToolbox are called.

https://github.com/SteveMacenski/slam_toolbox/blob/78af8485ebfd7d2b95e50188a341b86b4069e644/src/slam_toolbox_common.cpp#L65-L97